README.md

```diff
@@ -293,4 +295,157 @@ INFO:logger:Command docker exec cfy_manager cfy executions start --timeout 3000
 ## Requirements:
 
   * pytest
-  * pytest-logger, add this to your test runner configuration: `--log-cli-level debug -s -v`.
\ No newline at end of file
+  * pytest-logger, add this to your test runner configuration: `--log-cli-level debug -s -v`.
+
+
+# Ecosystem test CLI
+
+Ecosystem tests CLI introduced in order to improve blueprint testing and continuous development of blueprints.
+Moreover, it makes testing blueprints and plugins via CI tools very intuitive.
+
+The CLI has three commands:
+
+* prepare-test-manager
+* local-blueprint-test
+* validate-blueprint
+
+## prepare-test-manager command
+
+`prepare-test-manager` command responsible for uploading license, plugins and create secrets on the manager before test invocation.
+If the manager has all the assets needed for the test, you can skip this command.
+
+### Options:
+
+`l, --license TEXT` - Licence for the manager, should be either path 
+to licence file or base64 encoded licence string. Default: license.yaml
+
+`-s, --secret TEXT` - A secret to update on the manager, should be provided 
+as secret_key=secret_value. This argument can be used multiple times.
+
+ `-fs, --file-secret TEXT` - A secret to update on the manager, should be 
+ provided as secret_key=file_path. This argument can be used multiple times.
+
+`-es, --encoded-secret TEXT` - Base 64 encoded secret to update on the manager,
+should be provided as secret_key=secret_value_base_64_encoded. 
+This argument can be used multiple times.
+
+`-p, --plugin TEXT` - Plugin to upload before test invocation, should be
+provided as --plugin plugin_wagon_url plugin.yaml_url. This argument can be used
+multiple times.
+
+`--bundle-path PATH` - Plugins bundle tgz file path.
+
+`--skip-bundle-upload` - Specify --skip-bundle-upload for not uploading
+plugins bundle before the test. Default: False.
+
+`-c, --container-name TEXT` - Manager docker container name. Default: cfy_manager.
+
+`--yum-package TEXT` - Yum package to install on the manager container.
```

Comment:
Maybe we should add a note when certain args can be provided more than once?

Reply:
Done.